src/suiryoku/locomotion/process/locomotion.cpp

```diff
@@ -900,13 +931,13 @@ bool Locomotion::position_kick_range_pan_tilt(const keisan::Angle<double> & dire
     return true;
   }
 
-    // y movement
-  if (!precise_kick) left_kick = pan > 0.0_deg;
+  // y movement
+  left_kick = precise_kick ? left_kick : (pan > 0.0_deg);
```

Comment:
In my opinion, the previous style is better since what is the point of setting `left_kick = left_kick`?